example3/readme.md

```diff
@@ -0,0 +1,580 @@
+# Summary
+
+An experimental project to implement an OpenTelemetry Metrics API and SDK.
+
+Our goals are:
+
+- To discover/design a simpler approach to Metrics API
+- To explore different SDK implementation based on Metrics API design
+- To experiment with alternative designs/models/ideas for evaluation
+- To complement existing/current .NET runtime design
+- To maintain zero/minimal differences between OpenTelemetry specification and .NET runtime
+- To use this project as an informed starting base for discussion with wider OpenTelemetry community
+
+## Principles
+
+- .NET provides a way to pass datapoints (un-opinionated) in a Time Series.  With
+optional metadata / hints for downstream interpretation.
+
+- .NET may implement and deliver counters (derived from .NET MetricBase) with it's
+own Semantic. (i.e. Counter with Add(), ElapsedDuration as a IDisposable, etc...)
+
+- OTel may implement and deliver counters (derived from .NET MetricBase) with it's
+own Semantic. (i.e. UpCounter with Inc(), ValueRecorder with Record(), etc...)
+
+[Noah]: This principle worries me. I want .NET developers to treat .NET metrics as being 
+a single unified concept. Ie if I am writing a .NET library I want to intrument it:
+a) Without having any reference to an OT specific library
+b) Without mentally apportioning some metrics as being intended for OT usage and others
+are not intended for OT usage.

```

Comment:
The unifying concept needs to be the MetricBase.  

I think .NET will provide the basic set of "instruments" (i.e. A Sum and a Gauge) that will cover 95+ % of basic needs.  However, some semantics are opinionated, and people can take OTel API if they want those opinionated "instrumnents" (i.e. UpDownCounter, Summary, Histogram, etc...).  Plus, people and other 3P libraries will derived their own from MetricBase with it's own semantics.